source/posts/2017-02-16-migrating-snapci-to-gocd.html.markdown.erb

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Migrating from Snap CI to GoCD"
+type: post
+author: Suzie Prince, Product Manager, Snap CI
+excerpt: "Many Snap CI users are now looking for a replacement CI/CD tool. In the end of life announcement we recommended that Snap CI users take a look at GoCD and consider it as an alternative."
+title_tag_of_header: "Migrating from Snap CI to GoCD | GoCD Blog"
+meta_description: "Snap CI is going away. GoCD is the recommended replacement for your Continuous Delivery Pipelines"
+meta_keywords: "GoCD, continuous delivery, goforcd, open source, software, contribute to open source, agile continuous delivery, code contribution, Go"
+summary_image: "/assets/images/blog/migrating-snapci-gocd/snapci-migrating-gocd.jpg"
```

Comment:
Please see https://middlemanapp.com/basics/helper_methods/#asset-helpers when linking to any images.